### PR TITLE
Feat(TDP-3109): link hamburger and topnav

### PR DESCRIPTION
**Pull request by douglasmik**
Merge HamburgerMenu with TopNav component to create GlobalNav component
TDP-3109

### CODE REVIEW

File: packages/ts-newskit/src/components/global-nav/globalnav.stories.mdx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { boolean } from "@storybook/addon-knobs"
 import { NewsKitProvider } from 'newskit';
 import { TimesWebLightTheme } from '../../theme';
 import { GlobalNav } from './index.tsx';
+import data from '../fixtures/data.json';
 
 <Meta
   title="Newskit/Global Navigation"
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/global-nav/globalnav.stories.mdx:9]: if you add `parameters={{
    layout: 'fullscreen'}}` will remove extra padding around the story

File: .storybook/main.js [hunk 2]
```diff
@@ -9,7 +9,8 @@ module.exports = {
     "@storybook/addon-essentials",
     "@storybook/addon-knobs",
     "@storybook/addon-options",
-    "@storybook/addon-viewport"
+    "@storybook/addon-viewport",
+    "storybook-addon-paddings"
```
Comment (bibicollins1) [inline .storybook/main.js:13]: What is this doing?

Comment (douglasmik) [reply .storybook/main.js:13]: iin connection to your point above, this allows us to dynamically set padding in storybook

File: packages/ts-newskit/src/components/global-nav/hamburger-menu/HamburgerMenu.stories.mdx [hunk 3]
```diff
@@ -29,23 +29,29 @@ Hamburger Menu
...
+      <HamburgerMenu data={data} loggedIn={true} />
+    </ThemeProvider>
   </Story>
 </Canvas>
 
 Hamburger Menu Logged out -  Buttons available on XS and SM screens
 
 <Canvas>
   <Story name="HamburgerMenu Logged out">
+    <ThemeProvider theme={TimesWebLightTheme}>
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/global-nav/hamburger-menu/HamburgerMenu.stories.mdx:52]: While you're here hahah do you mind also adding this to the <Story name="HamburgerMenu - Data edit"> one below?

File: packages/ts-newskit/src/components/global-nav/hamburger-menu/NavigationList.tsx [hunk 4]
```diff
@@ -26,7 +26,7 @@ const NavigationList: React.FC<{
                 }}
               >
                 {item.items.map(i => (
-                  <>
+                  <Fragment key={`sub-${i.slug}`}>
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/global-nav/hamburger-menu/NavigationList.tsx:29]: 👍

File: .storybook/main.js (hunk 2 above)
Comment (bibicollins1) [reply .storybook/main.js:13]: Oh maybe I need to run a yarn to see the button for that couldn't see it. Will try now 👍

Comment (douglasmik) [reply .storybook/main.js:13]: Screenshot 2023-02-24 at 15 30 55

File: packages/ts-newskit/src/components/global-nav/hamburger-menu/HamburgerMenu.stories.mdx [hunk 5]
```diff
@@ -29,23 +29,29 @@ Hamburger Menu
 
 <Canvas>
   <Story name="Hamburger Menu">
-    <HamburgerMenu data={data} loggedIn={boolean('Is logged in', false)} />
+    <ThemeProvider theme={TimesWebLightTheme}>
+      <HamburgerMenu data={data} loggedIn={boolean('Is logged in', false)} />
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/global-nav/hamburger-menu/HamburgerMenu.stories.mdx:33]: Think you need to change the `loggedIn` here to `isLoggedIn` to reflect your change

**Pull request merged**